roles/openshift_monitor_availability/tasks/install.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Ensure openshift-monitor-availability project
+  oc_project:
+    state: present
+    name: openshift-monitor-availability
+    description: Openshift availability monitoring applications.
+
+- import_tasks: install_monitor_app_create.yaml
```

Comment:
@ironcladlou shouldnt it be enough just to create an additional task eg. `install_web_console_smoke_tests` and import it here ? Cause you mentioned that:
> Goal is to get mine merged and then accept followups in the form of new playbooks within a single role.

So not sure why we need a playbook for each monitoring app.

Reply:
> So not sure why we need a playbook for each monitoring app.

I think Dan means new task files?

Reply:
Yes, I misspoke, a new task (and supporting files) are what's  needed, NOT a new playbook. Sorry about that.

Reply:
Clarified this in the README